docs/reference/data-streams/data-streams.asciidoc

```diff
@@ -74,10 +74,8 @@ such as:
 [[data-streams-rollover]]
 == Rollover
 
-When you create a data stream, {es} automatically creates a backing index for
-the stream. This index also acts as the stream's first write index. A
-<<indices-rollover-index,rollover>> creates a new backing index that becomes the
-stream's new write index.
```

Comment:
With the migration API, this is no longer the case. I'm not sure the first two sentences were needed anyway.

Reply:
It is still true that a backing index is created automatically when you create a data stream with the Create Data Stream API. It's entirely up to you whether that needs to be mentioned here or not.

Reply:
I relocated these first two sentences to the "Set up a data stream" page with 800ae627f6a.